static/app/components/events/breadcrumbs/breadcrumbsDataSection.tsx

```diff
@@ -143,12 +144,13 @@ export default function BreadcrumbsDataSection({
   const hasViewAll = summaryCrumbs.length !== enhancedCrumbs.length;
 
   return (
-    <EventDataSection
+    <InterimSection
       key="breadcrumbs"
```

Comment:
we seem to be pretty inconsistent about passing in key in as a prop for `EventDataSection`. might be useful to just standardize that while we are making this change?

Reply:
Building off this - since we are already passing in key?/type `InterimSection` i wonder if we need a full additional prop (FoldSectionKey) for `InterimSection`. is there a reason we can't just have one enum for the sections and expand the local storage key to add in the additional "issue-details" prefix? it just seems a bit repetitive now

Reply:
Makes a lot of sense, didn't notice the overlap. I'll use the `type` key and change the enum a bit. Good call!